packages/amazonq/src/extensionShared.ts

```diff
@@ -36,6 +37,20 @@ export async function activateShared(context: vscode.ExtensionContext) {
     const contextPrefix = 'amazonq'
     globals.contextPrefix = 'amazonq.' //todo: disconnect from above line
 
+    // Avoid activation if older toolkit is installed
+    // Amazon Q is only compatible with AWS Toolkit >= 3.0.0
+    const toolkit = vscode.extensions.getExtension(VSCODE_EXTENSION_ID.awstoolkit)
+    if (toolkit) {
+        const toolkitVersion = semver.coerce(toolkit.packageJSON.version)
+        if (toolkitVersion && toolkitVersion.major < 3) {
+            await vscode.window.showInformationMessage(
+                `Amazon Q is not compatible with AWS Toolkit ${toolkit.packageJSON.version}. Please upgrade your AWS Toolkit extension to the latest version and then restart VS Code.`,
+                'OK'
```

Comment:
We should link to the extension page for toolkit in vscode.

Reply:
Added in latest commit

Reply:
I am not seeing the change

Reply:
It was in https://github.com/aws/aws-toolkit-vscode/pull/4711/commits/d651f8f743cd3504d0c31ff556a04aa969ed59b0. 

However, I reverted it because the PM document does not mention a extension page.